API/events/triggerEvent.m

```diff
@@ -108,25 +108,28 @@ function triggerEvent(eventType, varargin)
 end
 
 
-function [packedArray, counts] = packCellArray(cellArray, col)
+function [packedArray, dims] = packCellArray(cellArray, col)
     % Packs a specified column of a cell array with different sized arrays into a 
-    % single row array and an array of counts for each cell. For the example below 
-    % reflect will be [1, 2, 3, 4, 5, 6, 7] and nReflect will be [3, 4]
+    % single row array and an array of the dimensions for each cell. For the example 
+    % below reflect will be [1, 2, 3, 4, 5, 6, 7] and nReflect will be [3, 1, 4, 1]
     % 
     % [reflect, nReflect] = packCellArray({[1; 2; 3], [4; 5; 6; 7]}, 1);
```

Comment:
I think it would be clearer if the output names in the docstring and example were updated to match those of the function signature.